bin/bundle-extensions

```diff
@@ -1,39 +1,43 @@
 #!/usr/bin/env python
-
+"""Copy bundle extension files to the client/app/extension directory"""
 import os
-from subprocess import call
-from distutils.dir_util import copy_tree
-
-from pkg_resources import iter_entry_points, resource_filename, resource_isdir
-
+from pathlib2 import Path
+from shutil import copy
 
+from redash import create_app, extensions
 
 # Make a directory for extensions and set it as an environment variable
 # to be picked up by webpack.
-EXTENSIONS_RELATIVE_PATH = os.path.join('client', 'app', 'extensions')
-EXTENSIONS_DIRECTORY = os.path.join(
-    os.path.dirname(os.path.dirname(__file__)),
-    EXTENSIONS_RELATIVE_PATH)
-
-if not os.path.exists(EXTENSIONS_DIRECTORY):
-    os.makedirs(EXTENSIONS_DIRECTORY)
-os.environ["EXTENSIONS_DIRECTORY"] = EXTENSIONS_RELATIVE_PATH
-
-for entry_point in iter_entry_points('redash.extensions'):
-    # This is where the frontend code for an extension lives
-    # inside of its package.
-    content_folder_relative = os.path.join(
-        entry_point.name, 'bundle')
-    (root_module, _) = os.path.splitext(entry_point.module_name)
-
-    if not resource_isdir(root_module, content_folder_relative):
+extensions_relative_path = Path('client', 'app', 'extensions')
+extensions_directory = Path(__file__).parent.parent / extensions_relative_path
+
+if not extensions_directory.exists():
+    extensions_directory.mkdir()
+os.environ["EXTENSIONS_DIRECTORY"] = str(extensions_relative_path)
+
+# We need the app here for logging and to make sure the bundles have loaded.
+app = create_app()
```

Comment:
Why bundles need the `app` instance?

Reply:
This needs the app to trigger the `redash.extensions.load_bundles` function, alternatively we can call it manually without the app here.

Reply:
But why `load_bundles` needs `app`? Except for using the logger it doesn't seem to need it.

Reply:
Ah, no good reason other than the app logger. I can detach that.